evennia/commands/default/unloggedin.py

```diff
@@ -465,3 +465,64 @@ def func(self):
                 utils.get_evennia_version(),
             )
         )
+
+
+def _create_account(session, accountname, password, permissions, typeclass=None, email=None):
+    """
+    Helper function, creates an account of the specified typeclass.
+    """
+    try:
+        new_account = create.create_account(
+            accountname, email, password, permissions=permissions, typeclass=typeclass
+        )
+
+    except Exception as e:
+        session.msg(
+            "There was an error creating the Account:\n%s\n If this problem persists, contact an"
+            " admin." % e
+        )
+        logger.log_trace()
+        return False
+
+    # This needs to be set so the engine knows this account is
+    # logging in for the first time. (so it knows to call the right
+    # hooks during login later)
+    new_account.db.FIRST_LOGIN = True
+
+    # join the new account to the public channel
+    pchannel = ChannelDB.objects.get_channel(settings.DEFAULT_CHANNELS[0]["key"])
+    if not pchannel or not pchannel.connect(new_account):
+        string = "New account '%s' could not connect to public channel!" % new_account.key
+        logger.log_err(string)
+    return new_account
+
+
+def _create_character(session, new_account, typeclass, home, permissions):
```

Comment:
This looks like it's doing what `Account.create_character` does with some more reporting ...